src/random_number.py

```diff
@@ -0,0 +1,11 @@
+# Write a program that generates a random number.
+# Output:
+# The random number is: 4
+import random;
+
+def random_number() -> int:
+  return random.randint(0,400) 
+
+
+print("The random number id: " + str( random_number() ))
```

Comment:
questo lo metterei dentro un `if __name__ == "__main__"` https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do